src/components/audio-player/audio-player.stories.js

```diff
@@ -0,0 +1,155 @@
+
+					</div>
+
+					<div class="audio-player__body">
+						<div class="audio-player__headings">
+							<h1 class="audio-player__title">${model.track.title}</h1>
+							<h2 class="audio-player__sub-title">${model.track.subtitle}</h2>
+						</div>
+						<div class="audio-player__controls-wrapper">
+							<div class="audio-controls">
+								<div class="audio-controls__play-wrapper">
+									<button class="js-audio-player__play audio-controls__play">
+										<span class="js-play-icon audio-controls__play-icon"> ${playIcon()} </span>
+										<span class="js-pause-icon audio-controls__pause-icon"> ${pauseIcon()} </span>
+									</button>
+								</div>
+								<div class="audio-controls__time-controls js-audio-player__scrubbable-area">
+									<div class="js-audio-player__scrubbing-start-area audio-controls__scrubber-wrapper">
+										<canvas
+											width="10000"
+											height="6"
+											class="js-audio-player__progress-bar audio-player__progress-bar"
+										>
+										</canvas>
+									</div>
+									<div class="audio-controls__time-buttons">
+										<div class="audio-controls__back-controls">
+											<span class="js-audio-player__current-time">0:00</span>
+											<button
+												class="js-audio-player__seek-back-helper audio-controls__seek-back"
+											>
+												${rewindTenSecondsIcon()}
+											</button>
+										</div>
+										<div class="audio-controls__forward-controls">
+											<button
+												class="js-audio-player__seek-forward-helper audio-controls__seek-forward"
+											>
+												${forwardTenSecondsIcon()}
+											</button>
+											<span class="js-audio-player__remaining">0:00</span>
+										</div>
+									</div>
+								</div>
+
+								<audio
+									class="js-audio-player__audio audio-player__audio-element"
+									title="${model.track.title}"
+									style="width: 100%; height: 36px;"
+									controls
+								>
+									<source src="${model.track.audioFileURL}" />
+									<p>
+										Your browser doesn't support HTML5 audio. Here is a
+										<a href="${model.track.audioFileURL}"
+											>link to download the audio</a
+										>
+										instead.
+									</p>
+								</audio>
+							</div>
+						</div>
+					</div>
+				</div>
+				${model.track.transcript.length > 0 ? html `<div
+						class="audio-player__transcript-section js-audio-player__transcript-section"
+					>
+						<div
+							class="audio-player__transcript-wrapper js-audio-player__transcript-wrapper"
+						>
+							<div class="audio-player__transcript">
+								<p>${model.track.transcript}</p>
+							</div>
+						</div>
+						<a
+							href="#"
+							class="audio-player__transcript-toggle js-audio-player__transcript-toggle"
+						>
+							<span class="transcript__toggle-icon"> ${upCarrotIcon()}</span>
+							<spacn class="transcript__toggle-text js-transcript__toggle-text"> View Transcript </span>
```

Comment:
Couple lil things:

- `<spacn>` typo 🙃
- I believe this should be a `button` (interaction) not an `a` (navigational)?
- When switched to a `button` it should probably need a label or some other treatment besides just the upCarrot 🥕 to describe a) closing the drawer and b) which of the two states it's currently in